pkg/podres/mock_PodresourceClient.go

```diff
@@ -0,0 +1,76 @@
+package podres
+
+import (
+	context "context"
+
+	grpc "google.golang.org/grpc"
+
+	mock "github.com/stretchr/testify/mock"
+	podresourcesapi "k8s.io/kubernetes/pkg/kubelet/apis/podresources/v1alpha1"
+)
+
+// MockPodResourceClient is an autogenerated mock type for the PodResourceClient type
+type MockPodResourceClient struct {
+	mock.Mock
+}
+
+// List provides a mock function with given fields: ctx, in, opts
+func (_m *MockPodResourceClient) List(ctx context.Context, in *podresourcesapi.ListPodResourcesRequest, opts ...grpc.CallOption) (*podresourcesapi.ListPodResourcesResponse, error) {
+	_va := make([]interface{}, len(opts))
```

Comment:
is the underscore a known naming pattern for go/mocks?

Reply:
Same as above!